app/org/sagebionetworks/bridge/models/schedules/Schedule.java

```diff
@@ -60,24 +90,62 @@ public String getCronTrigger() {
     public void setCronTrigger(String cronTrigger) {
         this.cronTrigger = cronTrigger;
     }
-    public Long getStartsOn() {
+    public DateTime getStartsOn() {
         return startsOn;
     }
-    public void setStartsOn(Long startsOn) {
+    @JsonProperty("startsOn")
+    public void setStartsOn(DateTime startsOn) {
         this.startsOn = startsOn;
     }
-    public Long getEndsOn() {
+    public void setStartsOn(String startsOn) {
+        setStartsOn(DateTime.parse(startsOn));
+    }
```

Comment:
Should we have unit tests for the parsing setters for: setStartsOn(String), setEndsOn(String), setExpires(String), setDelay(String), setInterval(String)?